.ci-config/config.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
```

Comment:
please move this file to `.github/workflows/config.xml`
we do not need extra folder, .github/workflows is all about CI. 

same for patch.

Reply:
done